src/driver/packages.mli

```diff
@@ -36,16 +38,18 @@ type modulety = {
   m_intf : intf;
   m_impl : impl option;
   m_hidden : bool;
+  m_pkg_dir : Fpath.t;
```

Comment:
This `m_pkg_dir` is used in contextes where the pkg name is probably consistent but we don't need to take the risk.
```suggestion
  m_pkg_dir : string * Fpath.t;
```

Reply:
Fixed as well.